tap_facebook/__init__.py

```diff
@@ -37,14 +39,20 @@ def main():
         )
 
     # use
-    api = FacebookAdsApi.init(access_token=access_token, timeout=10)
+    try:
+        api = FacebookAdsApi.init(access_token=access_token, timeout=10)
+        user = User("me", api=api)
+        ad_accounts = user.get_ad_accounts(fields=["account_id", "id"])
```

Comment:
Can we put the rest of the synchronization function into the try block here?

Having the code be disjointed like this makes it harder to read.